data-raw/colour_palettes_2dii.R

```diff
@@ -25,37 +25,37 @@ palette_2dii_sector <- tibble::tribble(
 )
 
 palette_2dii_power <- tibble::tribble(
-         ~label,       ~hex,
-         "power", "#7A2701",
-         "power", "#a63603",
-         "power", "#e6550d",
-         "power", "#fd8d3c",
-         "power", "#fdbe85",
-         "power", "#ffd4ad"
+           ~label,      ~hex,
+        "coalcap", "#7A2701",
+         "oilcap", "#a63603",
+         "gascap", "#e6550d",
+     "nuclearcap", "#fd8d3c",
+       "hydrocap", "#fdbe85",
+  "renewablescap", "#ffd4ad"
 )
```

Comment:
Maybe protect the formatting with `# styler: off` and `#styler: on` around the tribbles.